docs/index.html

```diff
@@ -529,17 +529,29 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 			const oldIframe = iframe;
 			iframe = oldIframe.cloneNode();
 
+			iframe.style.display = 'none';
+
 			if ( hash && titles[ splitHash[ 0 ] ] ) {
 
+				iframe.onload = function () {
+
+					iframe.style.display = 'unset';
+
+				};
+
+				iframe.onerror = function () {
```

Comment:
I can confirm the white flash in Safari and that the PR fixes the issue!

Just one question: Why do we need an `onerror` event listener? What error situation triggers the callback?

Reply:
I was thinking with the `onerror` would cover cases such as 404 or network failure and show default browser pages for the iframe to show but I did test both those cases and they didn't seem to work.

Perhaps the `onerror` should be removed as there's no proven use case for it so far.

Reply:
> Perhaps the onerror should be removed as there's no proven use case for it so far.

Sounds good!

Reply:
done :smile: